### PR TITLE
Sidebar - submenu expand problem

### PR DESCRIPTION
**Pull request by opnsenseuser**
Fixes the problem that the submenu does not remain expanded if the focus remains on the menu item.

### CODE REVIEW

**Comment by opnsenseuser** [comment]
@fichtner  it would be really important to merge this. I'm sorry for the inconvenience
regards rené

File: src/opnsense/www/js/opnsense_theme.js [hunk 1]
```diff
@@ -51,7 +51,9 @@ $(document).ready(function () {
             var nextDiv = that.next('div');
 
             navigation.css('width', '415px');
-            if (nextDiv.not('in')) {
+            if (that.next('div').hasClass('in')) {
+                /* no action needed */
+            } else {
```
Comment (fichtner) [inline src/opnsense/www/js/opnsense_theme.js:56]: Shorter is better?

```suggestion
            if (!that.next('div').hasClass('in')) {
```

Comment (opnsenseuser) [reply src/opnsense/www/js/opnsense_theme.js:56]: ok

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by opnsenseuser** [comment]
@fichtner you never stop learning. I tested it. It works perfectly. Thanks for the tip

**Comment by fichtner** [comment]
@opnsenseuser after taking a closer look a5a109f9bf0b6c makes sense too. the variable is defined so the main if should use it too just in case the variable is modified later on.

**Pull request merged**